saleor/graphql/product/tests/test_product.py

```diff
@@ -4572,22 +4550,23 @@ def test_search_product_by_description_and_name(
 
 
 def test_sort_product_by_rank_without_search(
-    user_api_client, product_list, product, channel_USD, category, product_type
+    user_api_client, product_list, channel_USD
 ):
-    product_count = Product.objects.count()
-
     variables = {
         "sortBy": {"field": "RANK", "direction": "DESC"},
         "channel": channel_USD.slug,
     }
     response = user_api_client.post_graphql(SEARCH_PRODUCTS_QUERY, variables)
-    content = get_graphql_content(response)
-    data = content["data"]["products"]["edges"]
-    assert len(data) == product_count
+    content = get_graphql_content(response, ignore_errors=True)
+    assert "errors" in content
+    assert (
+        content["errors"][0]["message"]
+        == "Sorting by Rank is available only with searching."
```

Comment:
Tests are failing on cloud build. This error message doesn't match:
https://github.com/saleor/saleor/blob/cfdd78cf7fb4a6814be8266fd42276aebcf424f0/saleor/graphql/product/schema.py#L370